gui/control.js

```diff
@@ -45,12 +54,13 @@ function render() {
         accountsDiv.insertAdjacentHTML('beforeend', userCardString);
 
         // Deleting User Profile
-        document.getElementById(`switch-btn-${nickname}`).addEventListener('click', async function () {
-            logger.debug(`switching ${nickname}`)
-            const error = await PSS.switchAccount(nickname);
-            if (!error) return;
-            showToast("Error!", errorToMessage[error])
-        });
+        document.getElementById(`switch-btn-${nickname}`).addEventListener('click', ()=>{
+          switchAccountHandler(nickname);
+      }); 
+
+      document.getElementById(`nick-${nickname}`).addEventListener('dblclick', ()=> {
+          switchAccountHandler(nickname);
+      });
```

Comment:
if you are using arrow function you dont really need to use curly braces
```
document.getElementById(`switch-btn-${nickname}`).addEventListener('click', ()=>switchAccountHandler(nickname););

 document.getElementById(`nick-${nickname}`).addEventListener('dblclick', ()=>switchAccountHandler(nickname));
```
should work

Reply:
Done